documentation.md

```diff
@@ -15,7 +15,14 @@ xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx<br>
 
 Where x is either a number or a lowercase letter.<br>
 
-This key has a limit of requests you can make to the API (per hour?) and for small projects this is enough, but if you really want to track all busses around Luxembourg, you will have to ask them to increase your limit.<br>
+This key has a limit of requests you can make to the API, for small projects this is enough, but if you really want to track all busses around Luxembourg, you will have to ask them to increase your limit.<br>
+
+Default limits are:
+    Hourly: 500 requests
+    Daily: 5000 requests
+
+There also exists a monthly and yearly quota, but the defined request amount is quite generous.<br>
```

Comment:
:) I have a quota of 600'000 requests per day haha

Reply:
what can you do with 5'000?
Can you even request all stations in Luxembourg?

Reply:
ok, yes you can, there are 2786 stations in LUX

Reply:
so twice a day you can call them all

Reply:
Atm i'm polling only every 5 mins Lux2Bascharage, and Bascharage2Lux trains to measure the real delay's.
Anyway, last week i noticed a train, delayed 10 mins on mobiliteit.lu, while the api gave 29 mins.....
I missed to check in cfl app.

You already tested https://data.public.lu/en/docapi/ ?
Swagger api, maybe useful.